### ISSUE TITLE
Property 'cas.enable.redirect.guest.to.login' not working because of _CURRENT_SERVER_NAME_ token

### ISSUE THREAD

**Issue by drewwills** [uPortal-Project/uPortal#1409]
## Description

The new support for multiple CAS servers breaks the `cas.enable.redirect.guest.to.login` property.

Some configs (currently)...

```
portal.protocol.server.context=${portal.protocol}://_CURRENT_SERVER_NAME_${portal.context}
[...]
portal.login.url=${portal.protocol.server.context}/Login
[...]
org.apereo.portal.channels.CLogin.CasLoginUrl=${cas.protocol.server.context}/login?service=${portal.login.url}
```

## To Reproduce

Set `cas.enable.redirect.guest.to.login=true`.

**Comment by drewwills** [comment]
@mrapczynski, @ChristianMurphy, @jgribonvald

**Comment by jgribonvald** [comment]
@drewwills : what do you obtain as url ?
Maybe I missed somewhere to apply the decorator of this url. But in all case you have the possibility to set the property `portal.protocol.server.context` to override the default's one

**Comment by ChristianMurphy** [comment]
> what do you obtain as url ?

`_CURRENT_SERVER_NAME_`

**Comment by jgribonvald** [comment]
ok I missed to apply the decoration somewhere, so this isn't not too much complicated to apply, but have to find where ...

**Comment by jgribonvald** [comment]
Ok I see the problem in the bean init, so 2 choice:

**Comment by jgribonvald** [comment]
this should be solved by #1410
